Jenkinsfile.SensorboxTests

```diff
@@ -1,17 +1,50 @@
 #!groovy
 pipeline {
-        agent {
-                node {
-                    label 'HardwareSensorBox'
-              }
-         }
+     agent {
+        dockerfile {
+            filename 'Dockerfile.jenkins'
+            // 'docker build' would normally copy the whole build-dir to the container, changing the
+            // docker build directory avoids that overhead
+            dir 'docker'
+            // run on slave1 where nexus is connected to
+            label 'DeviceHardware'
+            // Pass the uid and the gid of the current user (jenkins-user) to the Dockerfile, so a
+            // corresponding user can be added. This is needed to provide the jenkins user inside
+            // the container for the ssh-agent to work.
+            // Another way would be to simply map the passwd file, but would spoil additional information
+            additionalBuildArgs '--build-arg USER_ID=$(id -u) --build-arg GROUP_ID=$(id -g)'
+            args "--net host -v /dev/bus/usb/:/dev/bus/usb --device /dev/kvm:/dev/kvm -v /var/local/container_shared/gradle/:/.gradle -v /var/local/container_shared/android-sdk:/usr/local/android-sdk -v /var/local/container_shared/android-home:/.android -v /var/local/container_shared/emulator_console_auth_token:/.emulator_console_auth_token -v /var/local/container_shared/analytics.settings:/analytics.settings"
```

Comment:
/dev/kvm is used by QEMU for the Android Emulator. So it shouldn't be necessary here.